libraries/GCS_MAVLink/examples/routing/routing.cpp

```diff
@@ -122,5 +123,10 @@ void loop(void)
     hal.scheduler->delay(1000);
 }
 
+/* dummy methods to avoid having to link against AP_Camera */
```

Comment:
> We've had a rule against things like CAMERA_DISABLED in library code (quite reasonably)

We do? Never heard of it. Yeah, if we want to support enable/disable of features for sure that rule needs to go away.